pkg/cmd/render/render.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cluster-authentication-operator/bindata"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type Permission os.FileMode
+
+const (
+	PermissionDirectoryDefault Permission = 0755
+	PermissionFileDefault      Permission = 0644
+)
+
+type AssetsFunc func(name string) ([]byte, error)
+
+type RenderOptions struct {
+	AssetOutputDir string
+	Assets         AssetsFunc
```

Comment:
How about using `resourceapply.AssetFunc` from library-go instead of defining a new func type?

Reply:
Good catch. I hadn't even considered that